src/applications/tests/unit_tests/signed_log_test.cpp

```diff
@@ -76,18 +76,22 @@ std::string StringWithDelta::get_current_state() const {
     return current_state;
 }
 
-void StringWithDelta::finalizeCurrentDelta(const persistent::DeltaFinalizer& finalizer) {
-    if(delta.size() == 0) {
-        dbg_default_trace("StringWithDelta: Calling finalizer with null buffer");
-        finalizer(nullptr, 0);
+size_t StringWithDelta::currentDeltaSize() {
+    return delta.size();
```

Comment:
Shouldn't this be mutils::bytes_size(delta)? The serialized size of a std::string includes an extra int for the length of the string, so bytes_size will be slightly bigger than string::size().